lib/fencing/st_output.c

```diff
@@ -412,11 +412,16 @@ stonith_event_text(pcmk__output_t *out, va_list args)
     const char *succeeded = va_arg(args, const char *);
     uint32_t show_opts = va_arg(args, uint32_t);
 
-    gchar *desc = stonith__history_description(event, full_history, succeeded,
-                                               show_opts);
+    if (out->is_quiet(out)) {
+        pcmk__formatted_printf(out, "%lld\n", (long long) event->completed);
```

Comment:
Are we sure all the other callers of stonith-event don't set quiet?

Reply:
stonith_admin calls pcmk__fence_history and sets out->quiet based on command line args.  It stands to reason that other command line tools would do the same.

Reply:
To expand upon this a little bit, the whole point of having separate behavior in this message if is_quiet is true was to keep compatibility with tools when moving them to formatted output.  I don't remember exactly which tool it was anymore, but something wanted just this one number printed out.

It appears the other command line tools that can call stonith-event are crm_mon (which does not appear to set out->quiet, weird) and crm_simulate (which does set it).  It's possible I'm missing one - this is kind of a tangle.

Reply:
How is crm_simulate calling it? It doesn't connect to the fencer.

If that's everything, I don't think there's any problem.

Reply:
> So I guess this is a change when stonith_admin is called with -H and -q?

Right, it actually looks like the entire purpose of is_quiet in the first place was to support these options to stonith_admin according to 94769e966d.  Then if you look at f0c484f102 you will see basically the same block in handle_history that has been moved and tweaked into its current state.

Reply:
> How is crm_simulate calling it? It doesn't connect to the fencer.

Ah okay.  It calls pcmk__simulate, which calls the cluster-status message, but does not pass the flags necessary for any of the fencing messages to be called.  So we're okay there.

Reply:
> To expand upon this a little bit, the whole point of having separate behavior in this message if is_quiet is true was to keep compatibility with tools when moving them to formatted output. I don't remember exactly which tool it was anymore, but something wanted just this one number printed out.

Right, but this will change the behavior for callers of "stonith-event" other than pcmk__fence_history(). We need to be confident those callers won't be adversely affected.